src/SFA.DAS.FindEmploymentSchemes.UnitTests/Web/Controllers/PagesControllerTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using FakeItEasy;
+using Xunit;
+using SFA.DAS.FindEmploymentSchemes.Web.Controllers;
+using SFA.DAS.FindEmploymentSchemes.Web.Models;
+
+
+namespace SFA.DAS.FindEmploymentSchemes.UnitTests.Web.Controllers
+{
+    public class PagesControllerTests
```

Comment:
It'd be good if we tested a negative case too, i.e. a Not Found response.

Reply:
Added a couple of Not Found tests